### PR TITLE
Dockerizing Services

### PR DESCRIPTION
**Pull request by edwinPato**
I am refactoring code and adding files for dockerizing the proyect.

### CODE REVIEW

File: CONVERTER_SERVICE/.env [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+CONVERTER_PATH_EXIFTOOL=exiftool
+OUTPUT_PATH=/app/resources/outputPath/
+UPLOAD_PATH=/app/resources/upload/
+VIDEO_PATH=/app/resources/video/
+AUDIO_PATH=/app/resources/audio/
+ZIP_PATH=/app/resources/zip/
+BASE_URL_CONVERTER="http://localhost"
+
+PORT_CONVERTER=4000
+IP_MONGO=192.168.1.106
```
Comment (serqiiiio) [inline CONVERTER_SERVICE/.env:11]: add last line

File: CONVERTER_SERVICE/Dockerfile [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+COPY package*.json ./
 
 RUN npm install
 
 COPY . .
 
-EXPOSE 3000
-
-CMD npm start
+CMD [ "npm", "start" ]
```
Comment (serqiiiio) [inline CONVERTER_SERVICE/Dockerfile:17]: add a last line

File: docker-compose.yml [hunk 3]
```diff
@@ -24,11 +24,27 @@ services:
...
     image: mongo:latest
     volumes:
       - ./db:/data
     ports:
       - "27018:27017"
+    networks: 
+      - my-network-name
 networks:
-  my-network-name:
+  my-network-name:
```
Comment (serqiiiio) [inline docker-compose.yml:50]: add last line

File: CONVERTER_SERVICE/Dockerfile (hunk 2 above)
Comment (edwinPato) [reply CONVERTER_SERVICE/Dockerfile]: Done!

File: docker-compose.yml (hunk 3 above)
Comment (edwinPato) [reply docker-compose.yml]: Done!

**Review by serqiiiio — APPROVED** [review]

**Pull request merged**